### DISCUSSION TITLE
v1.9.6

### DISCUSSION THREAD

**Discussion by vran-dev** [vran-dev/PrettyZoo#251] [General]
1. fix ui typo，contribute by @captjt 
2. remmber ui change
3. add windows zip installer

<hr /><em>This discussion was created from the release <a href='https://github.com/vran-dev/PrettyZoo/releases/tag/v1.9.6'>v1.9.6</a>.</em>

**Comment by vleyong** [comment]
Warning logs was printed when I installed on the linux by rpm?
Preparing...                          ################################# [100%]
Updating / installing...
   1:prettyzoo-1.9.6-1                ################################# [100%]
xdg-desktop-menu: file '/opt/prettyzoo/lib/prettyzoo-prettyZoo.desktop' does not exist
warning: %post(prettyzoo-1.9.6-1.x86_64) scriptlet failed, exit status 2

**Comment by blackmaxer** [comment]
Tip: Child process exited with code 1！
What happen..
When I click prettyZoo.exe..
